big_tests/tests/gdpr_SUITE.erl

```diff
@@ -202,22 +216,146 @@ retrieve_offline(Config) ->
               Alice, Config, "offline", ExpectedHeader, ExpectedItems)
         end).
 
-retrieve_pubsub(Config) ->
+retrieve_pubsub_payloads(Config) ->
     escalus:fresh_story(Config, [{alice, 1}], fun(Alice) ->
-            Node = pubsub_tools:pubsub_node(),
-            ItemId = <<"puczjorhendsindier">>,
-            pubsub_tools:publish(Alice, ItemId, Node, [{with_payload, true}]),
-            PepNS = <<"gdpr:pep">>,
-            PepItemId = <<"puczjorhendsap">>,
-            pubsub_tools:publish(Alice, PepItemId, {pep, PepNS}, []),
-
-            ExpectedHeader = ["node_id", "item_id", "payload"],
-            ExpectedItems = [
-                            ],
-            retrieve_and_validate_personal_data(
-              Alice, Config, "pubsub", ExpectedHeader, ExpectedItems)
+        Node1 = {_Domain, NodeName1} = pubsub_tools:pubsub_node(),
+        Node2 = {_Domain, NodeName2} = pubsub_tools:pubsub_node(),
+        {BinItem1, StringItem1} = item_content(<<"Item1Data">>),
+        {BinItem2, StringItem2} = item_content(<<"Item2Data">>),
+        {BinItem3, StringItem3} = item_content(<<"Item3Data">>),
+        {BinOther, StringOther} = item_content(<<"OtherItemData">>),
+        pubsub_tools:publish(Alice, <<"Item1">>, Node1, [{with_payload, BinItem1}]),
+        pubsub_tools:publish(Alice, <<"Item2">>, Node1, [{with_payload, BinItem2}]),
+        pubsub_tools:publish(Alice, <<"Item3">>, Node1, [{with_payload, BinItem3}]),
+        pubsub_tools:publish(Alice, <<"OtherItem">>, Node2, [{with_payload, BinOther}]),
+
+        ExpectedItems = [pubsub_payloads_row_map(NodeName1, "Item1", StringItem1),
```

Comment:
How about:
```erlang
ExpectedOuts = [
{NodeName1, "Item1", StringItem1},
{NodeName1, "Item2", StringItem1},
{NodeName1, "Item3", StringItem1},
{NodeName2, "OtherItem", StringOther},
],
ExpectedItems = [ pubsub_payloads_row_map(NN, I, SI) || {NN, I, SI} <- ExpectedOuts]
```

Reply:
not sure if it actually changes anything